charts/dbildungs-iam/seeding/dev/02_person.json

```diff
@@ -0,0 +1,32 @@
+{
+    "entityName": "Person",
+    "entities": [
+        {
+            "referrer": null,
+            "mandant": "023c68a8-83a8-487f-812e-e4662c9862e1",
+            "stammorganisation": null,
+            "vorname": "Manuell",
+            "familienname": "Mustermann",
+            "initialenfamilienname": null,
+            "initialenvorname": null,
+            "rufname": null,
+            "titel": null,
+            "anrede": null,
+            "namenssuffix": null,
+            "namenspraefix": null,
+            "sortierindex": null,
+            "geburt": {
+                "datum": "2023-08-26T20:28:01.000Z",
```

Comment:
Bitte Geburtsdatum rausnullen. Das sollen wir auch in Prod nicht speichern, dann sollten wir das im Test auch nicht machen